app/assets/stylesheets/application.css.scss

```diff
@@ -98,6 +98,24 @@ input {
   margin-top: 1%;
 }
 
+.field > input[type="email"], input[type="password"] {
+  border-radius: 4px;
+  border: 0px;
+  width: 30%;
+  margin-top: 1%;
+  margin-bottom: 1%;
+  height: 30px;
+}
+
+.actions > input[type="submit"] {
+  border-radius: 4px;
```

Comment:
Properties should be ordered border, border-radius, height, margin-bottom, margin-top, width